packages/web-runtime/src/container/bootstrap.ts

```diff
@@ -37,17 +37,17 @@ export const requestConfiguration = async (path: string): Promise<RuntimeConfigu
  * @param runtimeConfiguration
  * @param store
  */
-export const announceStore = async ({
+export const announceStore = ({
   vue,
   runtimeConfiguration,
   store
 }: {
   vue: VueConstructor
   runtimeConfiguration: RuntimeConfiguration
   store: Store<any>
-}): Promise<void> => {
-  await store.dispatch('loadConfig', runtimeConfiguration)
-  await store.dispatch('initAuth')
+}): Promise<any> => {
```

Comment:
we must guarantee that config is loaded before proceeding with initAuth. Thats not the case if we parallelize it with Promis.all

Reply:
LOAD_EXTENSION_CONFIG for example, external apps are able to hook into the auth process (not yet) for example publicFiles needsAuth

Reply:
ok, so `announceApplications` needs to be called after `announceStore`. I'll change that.

Reply:
Yes, the store can be used in the hooks of a application. And to be safe it needs to happen before that